src/components/BRB/BRBMainComponent.tsx

```diff
@@ -805,6 +813,7 @@ const IconMenu = styled.ul`
  */
 const NavigationMenuItem = styled.li`
   position: relative;
+  font-family: Glancyr;
```

Comment:
Fallback font should be mentioned: https://developer.mozilla.org/en-US/docs/Web/CSS/font-family